### PR TITLE
Adds editor config

### LINKED ISSUE #23: Add .editorconfig to the solution

### PR DESCRIPTION
**Pull request by Camz-1601**
closes #23

### CODE REVIEW

File: ECM3440.sln [hunk 1]
```diff
@@ -1,12 +1,20 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SoilMoistureSensor", "SoilMoistureSensor\SoilMoistureSensor.csproj", "{402007D2-6A0A-438D-B2FA-9515917A203F}"
+# Visual Studio Version 17
```
Comment (ghost) [inline ECM3440.sln:3]: this doesn't look right why do we have a vs 17 reference in our sln?

**Review by ghost — CHANGES_REQUESTED** [review]

File: ECM3440.sln (hunk 1 above)
Comment (Camz-1601) [reply ECM3440.sln:3]: We don't 5 head

**Review by ghost — APPROVED** [review]
I simply do not want to read the editorconfig file so I'm gonna trust the process 🙏

**Comment by Camz-1601** [comment]
> I simply do not want to read the editorconfig file so I'm gonna trust the process 🙏

Cheers lad

**Review by DavidScott96 — APPROVED** [review]

**Pull request merged**